app/views/kitten/components/heroes/hero.html.erb

```diff
@@ -2,19 +2,13 @@
   <div class="k-Grid__col k-Grid__col--12">
     <%= example 'Hero' do %>
       <div class="k-Hero k-Hero--onDark k-Hero--catBackground">
-        <div class="k-Hero__container">
-          <div class="k-Hero__text">
-            <h1 class="k-Hero__title">Yarn balls</h1>
-            <p class="k-Hero__subtitle">Infinite fun.</p>
-            <p class="k-Hero__subtitle">Until humans have to roll them back.</p>
-          </div>
-          <div class="k-Hero__action">
-            <img src="https://placehold.it/300x400/fff/fff"
-                 width="300"
-                 height="400"
-                 style="max-width: 100%"
-                 alt="" />
-            <p>Kitten after a day of playing whith yarn</p>
+        <div class="k-Hero__text">
+          <h1 class="k-Hero__title">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h1>
+          <p class="k-Hero__subtitle">Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis.</p>
+        </div>
+        <div class="k-Hero__action">
+          <div class="k-Card k-Card--light k-Card--padded k-Card--withVerticalPadding k-Card--rounded">
+            <img class="k-Hero__img" />
```

Comment:
Si tu veux illustrer la classe `.k-Hero__img`, on peut sans doute l'ajouter à `<img class="k-Hero__img" src="https://placehold.it/300x400/4a84ff/4a84ff" alt="" />`.